src/devices/ServoMotor/README.md

```diff
@@ -0,0 +1,69 @@
+﻿# Servo Motor
+
+## Summary
+This is a generic binding to control many servo motors using a hardware or software `PwmChannel`.  Servo motors are usually based on a signal frequency of 50Hz.  They also require a minimum/maximum pulse width to determine the position.  The pulse width is generally between 1 and 2 milliseconds, where 1ms is approximately 0 degrees, 1.5ms is the rest position, and 2ms is 180 degrees.  This information can be found in each servo motor's datasheet.  
```

Comment:
add enter after `## Summary` - I found markdown often breaks if you don't put enter before and after certain things like headers, tables and code blocks

Reply:
Gotcha.  I like that better (with space), but did a verify on other bindings.  We'll probably need to do a future scan to update those.

This is updated in next commit.